interop-testing/src/main/java/io/grpc/testing/integration/AbstractTransportTest.java

```diff
@@ -811,6 +811,17 @@ public AccessToken refreshAccessToken() throws IOException {
         authScope.contains(response.getOauthScope()));
   }
 
+  /** Sends an rpc to an unimplemented method on the server. **/
+  public void unimplementedMethod() {
+    UnimplementedServiceGrpc.UnimplementedServiceBlockingStub stub =
+        UnimplementedServiceGrpc.newBlockingStub(channel);
+    try {
+      stub.unimplementedCall(Empty.getDefaultInstance());
+    } catch (Throwable t) {
+      assertEquals(Status.UNIMPLEMENTED, Status.fromThrowable(t));
```

Comment:
We could maybe do both. It does seem the test case is overly-strict. I know if we get a 404 from a server the client will say it came from an HTTP 404. We can talk to @a11r to determine why it requires the description to be null.